cron.sh

```diff
@@ -1,10 +1,51 @@
 #!/bin/bash
 
-git config --global user.email "drew@fishtownanalytics.com"
-git config --global user.name "Hubcap"
+# ==
+# == Setup
+# ==
 
-curl https://raw.githubusercontent.com/fishtown-analytics/hubcap/master/hub.json > hub.current.json
-echo ${CONFIG} > config.json
-rm -rf git-tmp/ROOT
-python hubcap.py
+export RUN_DIR="$(pwd)"
+export GIT_TMP="$RUN_DIR/git-tmp"
 
+exit_routine() {
+  local -r exit_status="${1}"
+
+    if [ -d "${GIT_TMP}" ]; then
+        rm -rf "${GIT_TMP}"
+    fi
+
+  exit "${exit_status}"
+}
+
+trap 'exit_routine $?' EXIT
+
+set -o errexit
+set -o errtrace
+set -o nounset
+set -o pipefail
+
+# ==
+# == Main
+# ==
+
+# specified in Heroku's config variables
+export ENV="${ENV-development}"
+if [ "$ENV" = 'prod' ] || [ "$ENV" = 'test' ]; then
+    email='drew@fishtownanalytics.com'
+    user_name='dbt-hubcap'
```

Comment:
Last comment, and absolutely something we can address later:

Do we want this to keep showing up as Drew? We have a ["machine user"](https://docs.github.com/en/developers/overview/managing-deploy-keys#machine-users) we could borrow for this purpose: @FishtownBuildBot

I think that could be as simple as:
```
    email='buildbot@fishtownanalytics.com'
    user_name='FishtownBuildBot'
```
And swapping whatever Drew-specific SSH key is in Heroku, for a personal access token tied to the BuildBot account

Reply:
I'd like to address this later but yes, I have this in my Notion dev notes that are more expansive than what I've written here. Literally no disagreement and something I'd like to do as we continue to revamp these systems and add others.